portugues/colaborando.md

```diff
@@ -40,6 +40,6 @@ Vende seu PC?
 
 [NÃO](nvenda/nao.md)
 
-Um tijolo pesa 1 quilo mais meio tijolo. Quanto pesa 1 tijolo e meio?
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.